package.json

```diff
@@ -119,7 +103,9 @@
     "release-it": "^17.0.0",
     "sort-package-json": "^2.6.0",
     "typescript": "^5.2.2",
-    "typescript-eslint": "^8.7.0"
+    "typescript-eslint": "^8.7.0",
+    "vite": "^5.4.9",
```

Comment:
@NullVoxPopuli is this intentionally added? Why would we need it?

Reply:
yea, we import from vite in the vitest.config